modules/cnv-creating-vm.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-create-vms.adoc
+
+[id="cnv-creating-vm_{context}"]
+= Creating a virtual machine from the CLI
+
+The `spec` object of the VirtualMachine configuration file references
+the virtual machine settings, such as the number of cores and the amount
+of memory, the disk type, and the volumes to use.
+
+Attach the virtual machine disk to the virtual machine by referencing
+the relevant PVC `claimName` as a volume.
+
+[NOTE]
+====
+https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/[ReplicaSet]
+is not currently supported in {ProductName}.
+====
+
+See the Reference section for information about volume types and
+sample configuration files.
+
+.Domain settings
+|===
+|Setting | Description
+
+|cores
+|The number of cores inside the virtual machine. Must be a value greater than or equal to 1.
+
+|memory
+| The amount of RAM allocated to the virtual machine by the node. Specify the denomination with *M* for Megabyte or *Gi* for Gigabyte.
```

Comment:
s/allocated/that is allocated
~ Specify the denomination with *M* for Megabyte or *Gi* for Gigabyte.~ Specify a value in megabytes (M) or gigabytes (Gi). 
Are you sure these aren't megabits and gigabits?

Reply:
Changed to:

"Memory
|The amount of RAM that is allocated to the virtual machine by the node. Specify a value in *M* for Megabyte or *Gi* for Gigabyte."

I will query QE on bit vs. byte.